tests/test_masks.py

```diff
@@ -100,3 +100,9 @@ def test_rectmask():
     assert np.allclose(m.rectangular(2, 2, -3, 3, 5, 5), rect3)
     assert np.allclose(m.rectangular(2, 2, 3, -3, 5, 5), rect2)
     assert np.allclose(m.rectangular(2, 2, -3, -3, 5, 5), rect1)
+
+
+def test_empty_rectmask():
+    assert not np.all(m.rectangular(2, 2, 0, 3, 5, 5))
+    assert not np.all(m.rectangular(2, 2, 3, 0, 5, 5))
+    assert not np.all(m.rectangular(2, 2, 0, 0, 5, 5))
```

Comment:
`not np.all` means that there is at least one false-y element in the array, is it possible that you mean `not np.any(...)` instead, as all elements are supposed to be `False`?

Reply:
Yup, that's what I meant. Thanks!